### PR TITLE
Update README.md with a link to Authors and adding a section for the file Contributing

### PR DESCRIPTION
**Pull request by antoine2711**
Make a real link to Authors and add a section for the file Contributing.
Regards, A.

### CODE REVIEW

**Comment by wetneb** [comment]
Are the headers between the two sentences really needed?

**Comment by antoine2711** [comment]
> Are the headers between the two sentences really needed?

It's a work in progress. At term, I'd like to add a phrase summarizing the document in that section, but I'm looking at the structure first.

Would you like me to remove them?

Regards,
   Antoine

File: README.md [hunk 1]
```diff
@@ -51,4 +51,10 @@ was acquired by Google, Inc. in July 2010 and the product was renamed Google Ref
...
 community-supported product.
 
-See `AUTHORS.md` for the list of OpenRefine contributors.
+## Authors
+
+See [AUTHORS.md](./AUTHORS.md) for the list of OpenRefine contributors.
+
+## Contributing
+
+See [CONTRIBUTING.md](./CONTRIBUTING.md).
```
Comment (tfmorris) [inline README.md:60]: Perhaps we can simplify and combine along the lines of:

```suggestion
See [AUTHORS.md](./AUTHORS.md) for the list of OpenRefine contributors and [CONTRIBUTING.md](./CONTRIBUTING.md) for instructions on how to contribute yourseslf.
```

Comment (antoine2711) [reply README.md:60]: Great. Even better. Thanks. 

Regards, A.

**Pull request merged**